templates/search/results-en.html

```diff
@@ -22,179 +22,203 @@
 </form>
 
 <section class="results">
-	<h2 class="wb-inv">Search results</h2>
-	<p class="h4">[number] search results for "[search term]"</p>
+	<div id="query-summary">
+		<h2 class="h4">[number] search results for "[search term]"</h2>
```

Comment:
Please remove the `h4` class since the font size for this `h2` is defined through the CSS and must be consistent.